src/AzureExtension/Helpers/IconLoader.cs

```diff
@@ -12,7 +12,7 @@ public class IconLoader
     public static string GetIconAsBase64(string filename)
     {
         Log.Logger()?.ReportDebug(nameof(IconLoader), $"Asking for icon: {filename}");
-        if (!Base64ImageRegistry.ContainsKey(filename))
+        if (!Base64ImageRegistry.TryGetValue(filename, out var value))

```

Comment:
nit: Doesn't look like we are using `value` here, so we can probably discard it.  Or, we can in theory also use it by setting `value = ConvertIconToDataString(filename))` within the if and add that to the dictionary instead and change the return outside the if to be `value`.

Reply:
Changed to "_"